src/factory.css

```diff
@@ -143,8 +142,9 @@ button, .buttonStyle {
 
 #blockFactoryContent {
   height: 85%;
-  width: 100%;
+  width: 80%;
   overflow: hidden;
+  float:right;
```

Comment:
Look at absolute positioning with `right: 0px`. Float has many other odd effects that you probably don't mean.